packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.module.scss

```diff
@@ -150,32 +180,26 @@
 
     @include bpk-breakpoint-small-tablet {
       margin-bottom: bpk-spacing-md();
-
-      @include bpk-heading-1;
     }
 
     @include bpk-breakpoint-mobile {
-      @include bpk-hero-5;
-    }
-
-    @include bpk-breakpoint-small-mobile {
       @include bpk-heading-2;
     }
   }
 
   &__subheading {
     @include bpk-body-default;
 
-    @include bpk-breakpoint-small-tablet {
-      @include bpk-subheading;
+    @include bpk-breakpoint-tablet {
```

Comment:
Changed to match updated designs here: https://www.figma.com/file/irZ3YBx8vOm16ICkAr7mB3/Backpack?node-id=13148%3A15252